data/boarding missions.txt

```diff
@@ -24,6 +24,343 @@ mission "Assisting Merchant"
 	# What the destination is doesn't matter, because you always 'decline' this.
 	destination Earth
 
+mission "Assisting Merchant (Hello)"
```

Comment:
I don't think that variant assist missions for the same government are necessary, especially given that you aren't changing the random numbers or credits to account for the increased number of jobs and therefore increased chance of being offered the mission.